_layouts/page.html

```diff
@@ -70,6 +70,7 @@ <h1 id="top">{{page.title}}</h1>
           {{page.lead | markdownify}}
         </div>
         {% endif %}
+        {% include page-type-info-box.html %}
```

Comment:
Should these appear on the landing pages (i.e. /resources/)? I feel like the language it seems a little off here: 
![image](https://user-images.githubusercontent.com/18104884/220450421-093eec87-ff8c-4f0c-983b-8e0f5d5466a3.png)

Reply:
cc: @nguyen-tina

Reply:
They shouldn't appear on landing pages -- I would only include them in the "child" article pages.

Reply:
done!

WDYT about the design standards page? I almost removed it, but the orientation seems useful there.

Reply:
Do you mean [this page](https://federalist-1716bf9c-dd79-4d96-8285-6e56dc391b84.sites.pages.cloud.gov/preview/usdoj-crt/beta-ada/kk/675/related-type-icons/law-and-regs/design-standards/)? If so, I would say that the "ADA Standards for Accessible Design" page is actually a child page, so it should remain!